app.js

```diff
@@ -1,6 +1,34 @@
-import { name } from './config';
-import { User, Product } from './models';
+import path from 'path';
+
+import {
+    name
+} from './config';
+import {
+    EVENT_TYPE
+} from './src/async/constants';
+
+import {
+    User,
+    Product
+} from './src/models';
+
+import DirWatcher from './src/async/dirwatcher/DirWatcher';
+import CsvImporter from './src/async/importer/CsvImporter';
 
 console.log(name);
 new User();
 new Product();
+
+const DATA_PATH = path.resolve(__dirname, 'data');
+
+const dirWatcher = new DirWatcher();
+const importer = new CsvImporter();
+
+dirWatcher
+    .watch(DATA_PATH, 1000);
+
+dirWatcher
+    .on(EVENT_TYPE.DIRWATCHER_CHANGED, async (path) => {
+        const data = await importer.readDir(path);
```

Comment:
As far as I understand, currently it will import all csv files from `DATA_PATH` every time smth changes in that directory, while it should import each uniq file only once:

> Note: every CSV file in a directory should be processed only once.